wlm-ui/src/MainPage/Channel/Channel.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+
+import { ChannelInfo } from '../../store/slices/channel/channel';
+import './Channel.scss'
+
+interface IProps extends ChannelInfo {
+    onClickUse: () => void;
+    onClickSetExposure: (exposure: number) => void;
+    onClickSetPeriod: (period: number) => void;
+};
+
+const Channel = (props: IProps) => {
```

Comment:
Is it common to name an HTML element like this? Is there no risk of confusion with interfaces like `ChannelInfo`?